### ISSUE TITLE
ABF file reading problem

### ISSUE THREAD

**Issue by AdityaAsopa** [BhallaLab/Asymmetries-Analysis-Scripts#2]
Locating which bytes correspond to what information by reading through the whole binary files byte-by-byte.

**Comment by dilawar** [comment]
https://github.com/swharden/pyABF/blob/master/docs/advanced/abf-file-format/readme.md

This will be of great help.

⁣Sent from handheld device. Please pardon brevity and typos.

Get BlueMail for Android ​

**Comment by AdityaAsopa** [comment]
Yes, I saw. I am using this info from pyABF to implement it in MATLAB.
Thanks :)
